### ISSUE TITLE
Department techfab's search input field is laggy

### ISSUE THREAD

**Issue by Inari-Whitebear** [tgstation/tgstation#80797]
## Issue Summary

When you use the search box of the techfab it behaves laggily.
Like if you quickly type `abcd`, it'll type `abc`, then reset to `ab`, then adds the d, so you end up with `abd` instead of `abcd` and things like that.
Feels like an odd thing of where it starts the search and as part of that resets the text field or so.

## Round ID:

All recent manuel rounds

## Reproduction:

1. Open techfab
2. quickly type in your search
3. see it being laggy, lght instead of light, etc

**Comment by newfren** [comment]
this also happens for the pda, a little

**Comment by jlsnow301** [comment]
SHOULD feel much more responsive after #80720, which I assume was not on live at the time of this issue's creation. tested it on manuel live with another admin and it felt fine. closing this as completed, feel free to reopen if it isn't resolved.

**Comment by necromanceranne** [comment]
This has not been fixed. There still appears to be the same laggy input. I think part of the reason might be due to icons being loaded as a new letter is being input. When nothing actually loads, an input isn't missed.

**Comment by necromanceranne** [comment]
Sorry, this is fixed for techfabs. It isn't fixed for autolathes. Since this is technically fixed I'll close this and make a new issue.
